client.go

```diff
@@ -1019,3 +1042,329 @@ func (c *Client) ListSnapshots(container string) ([]string, error) {
 
 	return names, nil
 }
+
+/*
+ * return string array representing a container's full configuration
+ */
+func (c *Client) GetContainerConfig(container string) ([]string, error) {
+	st, err := c.ContainerStatus(container)
+	var resp []string
+	if err != nil {
+		return resp, err
+	}
+
+	profiles := strings.Join(st.Profiles, ",")
```

Comment:
I don't think so, bc ppl may then cut-paste into a profile apply, and spaces are not ok there.